GRDB/Core/Database.swift

```diff
@@ -307,7 +307,7 @@ extension Database {
                 db.configuration.trace!(sql)
             }, dbPointer)
         #else
-            if #available(iOS 10.0, OSX 10.12, watchOS 3.0, *) {
+            if #available(iOS 10.0, OSX 10.12, watchOS 3.0, tvOS 10.0, *) {
```

Comment:
This is important, thanks. Now, did you check if other availability checks in GRBD should be updated as well?

Reply:
This was the only change needed to get my demo code working. I’ll check the other availability macros and make appropriate changes.